components/SupportModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Image from 'next/image';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Typography,
+  Button,
+  Stack,
+  IconButton,
+  Box,
+} from '@mui/material';
+import { X } from 'react-bootstrap-icons';
+
+interface SupportModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const supportData = {
+  imageSrc: '/images/nzc_img.png',
+  title: 'Welcome to NetZeroPaths',
+  content:
+    'Thanks for joining us on this journey towards a sustainable future for your city. Here are three quick tips to help you get started!',
+  getStartedUrl: '/welcome',
+};
+
+const SupportModal: React.FC<SupportModalProps> = ({ open, onClose }) => {
```

Comment:
I remember reading a while ago that using `React.FC` has no benefits and is more verbose, do you have any thoughts on this? I can try to dig up where I read it

Reply:
> React.FC is unnecessary: it provides next to no benefits and has a few downsides. (See below.)
https://github.com/facebook/create-react-app/pull/8177

I don't have a super strong opinion on this, but perhaps for consistency with can stick with the `const SupportModal = ({ open, onClose }: SupportModalProps) => { ` approach for consistency and simplicity

Reply:
Yes, sure. Good to know, thanks